public/src/pages/main/Footer.tsx

```diff
@@ -109,6 +109,14 @@ export const Footer: FC = () => {
                 <br />
                 <strong>Email:</strong> info@example.com
                 <br />
+                <strong>Technical info:</strong>{' '}
```

Comment:
Let's consider a more realistic scenario where an e-commerce application includes a link to the Privacy policy in PDF.

```diff
-    { name: 'Privacy policy', url: '/', icon: 'bx-chevron-right' }
+    { name: 'Privacy policy', url: 'https://takemeover-bright.s3.amazonaws.com/privacy-policy.pdf', icon: 'bx-chevron-right' }
  ];

```